packages/react-charting/src/components/SankeyChart/SankeyChart.styles.ts

```diff
@@ -1,4 +1,5 @@
 import { ISankeyChartStyleProps, ISankeyChartStyles } from './SankeyChart.types';
+import { HighContrastSelectorBlack } from '@fluentui/react/lib/Styling';
```

Comment:
The sankey hover callout looks very different than the hover callout in other charts

Reply:
This is fixed and callout is now similar to other charts.